dashboard/lib/pd/jot_form/select_question.rb

```diff
@@ -70,7 +70,11 @@ def get_value(answer)
 
       # @override
       def type_specific_summary
-        {options: options}
+        {
+          options: options,
+          allow_other: allow_other,
+          other_text: other_text
+        }
```

Comment:
This will require updating some tests, [here](https://github.com/code-dot-org/code-dot-org/blob/3f077edf563c9686ef0153771fa11f6c6760b25c/dashboard/test/lib/pd/jot_form/select_question_test.rb#L103-L109) and [here](https://github.com/code-dot-org/code-dot-org/blob/3f077edf563c9686ef0153771fa11f6c6760b25c/dashboard/test/lib/pd/jot_form/form_questions_test.rb#L107-L126).

Reply:
Are we using `allow_other` anywhere? Is it needed in the summary?

Reply:
Will remove